tic-tac-toe/src/scripts/storage-controller.js

```diff
@@ -0,0 +1,12 @@
+const storageName = "ticTacToe"
+let gameData = localStorage.getItem(storageName);
```

Comment:
It's const as far as I understand :)

Reply:
This's true. I fixed this.
 ```suggestion
const storageName = "ticTacToe"
const gameData = localStorage.getItem(storageName);
```